### PR TITLE
Allow meters in text mode to span empty neighbors to the right

### PR DESCRIPTION
**Pull request by cgzones**
Closes: #484

### CODE REVIEW

**Review by natoscott — APPROVED** [review]
LGTM - one tiny possible nit: in some of the pointer casts are using whitespace before the asterix where previously/typically there was/is none - e.g.  "(const Object \*)" vs "(const Object\*)"

File: Header.c [hunk 1]
```diff
@@ -150,35 +150,71 @@ void Header_reinit(Header* this) {
...
+      int height = pad;
+      for (int j = 0; j < Vector_size(meters); j++) {
+         const Meter* meter = (const Meter *) Vector_get(meters, j);
+
+         if (height >= curHeight + curMeter->h)
+            break;
+
+         height += meter->h;
+         if (height <= curHeight)
+            continue;
```
Comment (BenBE) [inline Header.c:198]: I'm not sure this successfully catches, if two meters are offset by a few rows.

```
+---+---+
|   | 3 |
+ 1 +---+
|   |   |
+---+ 4 +
| 2 |   |
+---+---+
```

Question related to when checking 1 if non-blank 4 is caught?
Also what of 2 if 4 were even one line higher?

**Comment by cgzones** [comment]
> in some of the pointer casts are using whitespace before the asterix where previously/typically there was/is none - e.g. "(const Object *)" vs "(const Object*)"

Dropped the blanks

File: Header.c (hunk 1 above)
Comment (cgzones) [reply Header.c:198]: I tested several cases and all seems to work.

**Pull request merged**